### ISSUE TITLE
Stop funktion does not work

### ISSUE THREAD

**Issue by dvovla** [marc-gist/node-red-contrib-countdown#4]

**Comment by MKaiser96** [comment]
I agree with you. I had to go back to version 1.4.1. This version works fine.

**Comment by pow4all** [comment]
Same here, stop function is not working in 1.4.2

Edit: I figured it out. If you send a payload of 0 or false and the topic 'control' it isn't working. If you send the payload of 0 or false without a topic it is working. Good to know.

**Comment by jeffreychausse** [comment]
Same here! Workaround proposed by @pow4all does not seem to work either.

I don't believe it is related to this issue, but I noticed a typo error at line 101.

`var remainingTicksMsg = { "payload": ticks, "cancled": cancel };`

I guess you meant **canceled** instead of **cancled**.

**Comment by wedlock** [comment]
Stop function in node red dont work.

**Comment by bierre2000** [comment]
I confirm it's not working neither here.
Even with the given exemple on the node page, timer won't stop.

**Comment by pal2110** [comment]
Got hit with the same problem. Quick skim through the code and it appears a `msg.topic: control`, and a `msg.payload: cancel` will get the timer to stop.

**Comment by zsiti** [comment]
> Got hit with the same problem. Quick skim through the code and it appears a `msg.topic: control`, and a `msg.payload: cancel` will get the timer to stop.

Indeed, "cancelling" the timer might be a functional workaround for some use-cases, but actually there is a difference between "stopping" and "cancelling" the timer.

The "stopping" function stops the timer and emits a "Timer Off" payload (if configured on the node) on the first output of the node, while the "cancel" function just stops the timer _without emitting_ any output.

As @MKaiser96 has stated in this thread above: if you downgrade from 1.4.2 to 1.4.1, the stopping function will work as intended.

**Comment by linker3000** [comment]
Any chance of this being fixed in the release version?

**Comment by frankverheyen** [comment]
When is this problem going to be fixed!!!! I am using this countdown node very much!

**Comment by Graefer** [comment]
Not working
